Backend/src/main/java/idealab/api/repositories/EmployeeRepo.java

```diff
@@ -1,11 +1,21 @@
 package idealab.api.repositories;
 
-import idealab.api.model.Employee;
+import java.util.List;
+
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
+import idealab.api.model.Employee;
+
 public interface EmployeeRepo extends CrudRepository<Employee, Integer> {
     Employee findEmployeeById(Integer id);
     Employee findEmployeeByUsernameEquals(String username);
     Employee findEmployeeByUsername(String username);
+
+    /**
+     * Gets a list of all users but only returns name, username, and role.
+     */
+    @Query("select firstName, lastName, username, role from Employee")
+    List<Object[]> findAllSimple();
```

Comment:
rather than return an Object array, can you create a new return object under DTO? This will also improve the response class by changing `List<Map<String,String>>` to `List<NewDTO>`

Reply:
Yes! Are you saying to essentially create another model class without linking it to the Sql tables?
I was also curious if it were possible to extend a class with those models. I’d essentially want to create an EmployeeBasic and then Employee that extends EmployeeBasic. Do you have any resources for how to do that when they are creating tables?

Reply:
@hannahkamundson So in general, I think it's bad practice to extend a table. I was thinking a separate DTO class that's not coupled to the table with only the fields you need.